base/uk.ac.stfc.isis.ibex.ui.nicos/src/uk/ac/stfc/isis/ibex/ui/nicos/NicosView.java

```diff
@@ -146,7 +154,52 @@ public void handleEvent(Event e) {
                 txtOutput.setTopIndex(txtOutput.getLineCount() - 1);
             }
         });
-
+        
+        Label lblQueuedScripts = new Label(parent, SWT.NONE);
+        lblQueuedScripts.setText("Queued scripts:");
+            
+        Composite tableComposite = new Composite(parent, SWT.NONE);
+        tableComposite.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true, 1, 1));
+        
+        TableColumnLayout tableColumnLayout = new TableColumnLayout();
+        tableComposite.setLayout(tableColumnLayout);
+        
+        final TableViewer tableViewer = new TableViewer(tableComposite, SWT.FULL_SELECTION | SWT.BORDER | SWT.NO_SCROLL | SWT.V_SCROLL);        
+       
+        tableViewer.getTable().setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true, 1, 1));
+        tableViewer.getTable().setToolTipText("");
+        tableViewer.getTable().setLinesVisible(true);
+        
+        TableViewerColumn tableViewerColumn = new TableViewerColumn(tableViewer, SWT.CENTER);
+        tableColumnLayout.setColumnData(tableViewerColumn.getColumn(), new ColumnWeightData(100, 0, false)); 
+        tableViewerColumn.setLabelProvider(new OwnerDrawLabelProvider() {
+        	// Using our own drawer means that we can write text to cover multiple lines
```

Comment:
It feels like this could be useful elsewhere. Can it be extracted as a separate class?